src/Form/Form.js

```diff
@@ -120,15 +120,15 @@ class Form {
         // If the user has changed the value, we prompt to update the stored creds
         this.shouldPromptToStoreCredentials = true
 
-        this.execOnInputs(this.removeInputHighlight, dataType)
+        this.execOnInputs((input) => this.removeInputHighlight(input), dataType)
     }
 
     removeInputDecoration (input) {
         removeInlineStyles(input, getIconStylesBase(input, this))
         input.removeAttribute(ATTR_AUTOFILL)
     }
     removeAllDecorations () {
-        this.execOnInputs(this.removeInputDecoration)
+        this.execOnInputs((input) => this.removeInputDecoration(input))
```

Comment:
Need to make sure these functions are called with the right context bound as they use `this` internally